tables/tr-g2.ctb

```diff
@@ -156,74 +240,200 @@ digit 9 247
 
 # multind 56-6 letsign capsign
 
-# Latin Harfleri
+# Accented Letters
```

Comment:
What concerns me a bit is that the grade 1 table is based on UEB, but the grade 2 isn't and takes definitions from the German table.

Reply:
@bertfrees wrote following he's review comment part:
> What concerns me a bit is that the grade 1 table is based on UEB, but the
> grade 2 isn't and takes definitions from the German table.

With original in 2017 Arend amend why designed turkish grade 1 braille system with ueb system, I don't known unfortunately. Technicaly and interests level I tried to look up the peculiarities of the Braille system in Turkish last week.
If you have guessed correctly, the Turkish Braille system is based on a mixed system, which includes Latin letters, accented characters based on the German language, but also some French and Irish characters. So a real mixed salad, at least according to the Wikipedia entry last modified in 2023:
[Turkish Braille](https://en.wikipedia.org/wiki/Turkish_Braille)
I not founds other good informations this topic related with helps understanding turkish Braille more advanced level or technical level to we detailed understanding the turkish level braille systems differences.